libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DateRangeHint.tsx

```diff
@@ -0,0 +1,28 @@
+// (C) 2025 GoodData Corporation
+
+import React from "react";
+import { IntlShape } from "react-intl";
+
+import { getLocalizedDateFormat } from "../utils/FormattingUtils.js";
+
+import { DATE_INPUT_HINT_ID, TIME_INPUT_HINT_ID } from "./types.js";
+
+export const DateRangeHint: React.FC<{
+    dateFormat: string;
+    isTimeEnabled: boolean;
+    intl: IntlShape;
+}> = ({ dateFormat, isTimeEnabled, intl }) => (
+    <div className="gd-date-range__hint">
+        <div id={DATE_INPUT_HINT_ID}>
```

Comment:
relying on fact that this component is rendered only once. If taken purely as reusable component which can be multiple times on page you should take ID as prop to avoid duplicates in page

Reply:
good point, fixed